### PR TITLE
Update JFR image to 220615_7db71b2

### PR DESCRIPTION
**Pull request by alxsap**
### Description

Update to latest JFR [220615_7db71b2](https://github.com/SAP/stewardci-jenkinsfilerunner-image/releases/tag/220615_7db71b2)

### Submitter checklist

- [x] Change has been tested (on a back-end cluster)
- [x] (If applicable) Jira backlog item ID added to the PR title and the [changelog.yaml] entry
- [x] [changelog.yaml] entry with upgrade notes is prepared and appropriate for the audience affected by the change (users or developer, depending on the change)
- [x] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
    - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)

In case dependencies have been updated:
- [x] Links to external changelogs, since the last release of our component, added to the [changelog.yaml] entry (description).
- [ ] Changelogs read thoroughly, potential impact described, upgrade notes prepared (if necessary)
- [ ] Check if dependency updates affect our semantic version increment.

### Reviewer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There is at least 1 approval for the pull request and no outstanding requests for change
- [ ] All voter checks have passed
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] The Pull Request title is understandable and reflects the changes well
- [ ] The Pull Request description is understandable and well documented
- [ ] [changelog.yaml] entry for this Pull Request has been added
    - [ ] Changelog entry contains all required information
    - [ ] 'Upgrade notes' are documented in changelog.yaml (if required)

[changelog.yaml]: https://github.com/SAP/stewardci-core/changelog.yaml
[releases]: https://github.com/SAP/stewardci-core/releases

### CODE REVIEW

File: charts/steward/values.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ pipelineRuns:
     elasticsearch:
       indexURL: ""
   jenkinsfileRunner:
-    image: "stewardci/stewardci-jenkinsfile-runner:220215_5d89c43"
+    image: "stewardci/stewardci-jenkinsfile-runner:220525_0a71212"
```
Comment (alxsap) [inline charts/steward/values.yaml:86]: Image has issues: https://github.com/SAP/stewardci-core/issues/327

File: changelog.yaml [hunk 2]
```diff
@@ -52,6 +52,13 @@
   date: TBD
   changes:
 
+    - type: security
+      impact: patch
+      title: Update JFR to 220525_0a71212
+      description: |-
+        Update to latest JFR [220525_0a71212](https://github.com/SAP/stewardci-jenkinsfilerunner-image/releases/tag/220525_0a71212)
```
Comment (alxsap) [inline changelog.yaml:59]:
```suggestion
      title: Update JFR to 220615_7db71b2
      description: |-
        Update to latest JFR [220615_7db71b2](https://github.com/SAP/stewardci-jenkinsfilerunner-image/releases/tag/220615_7db71b2)
```

File: charts/steward/values.yaml (hunk 1 above)
Comment (alxsap) [reply charts/steward/values.yaml:86]:
```suggestion
    image: "stewardci/stewardci-jenkinsfile-runner:220615_7db71b2"
```

File: changelog.yaml [hunk 3]
```diff
@@ -52,6 +52,15 @@
...
   changes:
 
+    - type: security
+      impact: patch
+      title: Update JFR to 220615_7db71b2
+      description: |-
+        Update to latest JFR [220615_7db71b2](https://github.com/SAP/stewardci-jenkinsfilerunner-image/releases/tag/220615_7db71b2)
+      upgradeNotes: |-
+        If you use an image cache you might want to update the image there as well.
+      pullRequestNumber: 326
```
Comment (alxsap) [inline changelog.yaml:62]:
```suggestion
      pullRequestNumber: 326
      jiraIssueNumber: 1640
```

**Review by romanisb — APPROVED** [review]

**Pull request merged**